js/src/api/local/accounts/account.js

```diff
@@ -68,21 +70,23 @@ export default class Account {
   }
 
   decryptPrivateKey (password) {
-    return keythereum.recover(Buffer.from(password), this._keyObject);
+    return decryptPrivateKey(this._keyObject, password);
   }
 
-  static fromPrivateKey (persist, key, password) {
-    const iv = keythereum.crypto.randomBytes(16);
-    const salt = keythereum.crypto.randomBytes(32);
-
-    // Keythereum will fail if `password` is an empty string
-    password = Buffer.from(password);
+  changePassword (key, password) {
+    return createKeyObject(key, password).then((keyObject) => {
+      this._keyObject = keyObject;
 
-    const keyObject = keythereum.dump(password, key, salt, iv);
+      this._persist();
+    });
+  }
 
-    const account = new Account(persist, { keyObject });
+  static fromPrivateKey (persist, key, password) {
```

Comment:
All the actual logic for this is in Keythereum, and the main issue here is that the tests take a while and can timeout in CI :|.